deployment/sklearn/sklearn-census-income-prediction.ipynb

```diff
@@ -0,0 +1,617 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8lCXP3A__nPN"
+      },
+      "source": [
+        "# Census Income Prediction (scikit-learn)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rjbaCPG4_vjE"
+      },
+      "source": [
+        "## 1. Dependencies"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EFsTgnMe_w9i"
+      },
+      "source": [
+        "This notebook has been tested with **Python 3.8.16** and the following package versions:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "YYZpIDPcky8c"
+      },
+      "outputs": [],
+      "source": [
+        "%%capture\n",
+        "!pip install verta==0.21.1\n",
+        "!pip install wget==3.2"
+      ]
```

Comment:
might want to include `pip install sklearn` as well to cover the only remaining module that is not Python-native.

Reply:
Done.